covasim/webapp/index.html

```diff
@@ -12,20 +12,19 @@
   <link type="text/css" rel="stylesheet" href="https://unpkg.com/bootstrap-vue@2.11.0/dist/bootstrap-vue.min.css"/>
   <link type="text/css" rel="stylesheet" href="assets/themify-icons.css"/>
```

Comment:
Do we still need this if you have the vendor.css?

Reply:
> Do we still need bootstrap-vue.css if you have the vendor.css?

Reply:
Removed. It is now moved to vendor css.